README.md

```diff
@@ -137,21 +139,19 @@ graph TD
         F3 -->|Persist results| G3
     end
 
-    subgraph "Glue Job Mode"
-        A2[Schedule triggers glue job]
-        B2{{"Some sales are already flagged"}}
-        C2[Ingest data for unflagged sales]
-        D2[Run flagging model within glue job]
-        E2[Write sales data to sale.flag]
-        F2[Join flags to<br>default.vw_pin_sale]
-        G2[Save results to S3 with unique run ID]
-
-        A2 -->|Trigger| B2
-        B2 -->|Process new sales| C2
-        C2 -->|Run model| D2
-        D2 -->|Output flags| E2
-        E2 -->|Join data| F2
-        F2 -->|Persist results| G2
+    subgraph "Manual Update (New Sales Only) Mode"
+        A4{{"Flag only new sales"}}
```

Comment:
**[Nitpick, blocking]** Quick change: Looks like we need to swap out the `style B2` block on line 171 so that it says `style A4` and highlights this node, right?

Reply:
Right, good catch